Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/configuration/AdminGUIManagementPortDescriptor.java

```diff
@@ -0,0 +1,20 @@
+package gov.hhs.fha.nhinc.configuration;
+
+import gov.hhs.fha.nhinc.adminguimanagement.AdminGUIManagementPortType;
+import gov.hhs.fha.nhinc.messaging.service.port.SOAP12ServicePortDescriptor;
+
+public class AdminGUIManagementPortDescriptor extends SOAP12ServicePortDescriptor<AdminGUIManagementPortType> {
```

Comment:
Can do. I mentioned this on the JIRA ticket as I was under the assumption we would have a dedicated ticket to moving everything else over instead of doing it all under the Dashboard ticket.

Reply:
Done.